Mtneer/mushu
# Styling

# Description
Incorporated styling from bulma.

## Type of change

# Testing Instructions
- There are no new features to be tested. You can run the app to ensure everything still loads and appears as expected.
- pull down new branch. Start app. Login.
- you may need to run npm install from the client directory in order to install dependencies such as fontawesome and bulma

# Checklist:
